azure-pipelines.yml

```diff
@@ -43,6 +43,20 @@ stages:
       parameters:
         MGMT_BRANCH: ""
 
+  - job: dependency_check
+    displayName: "Dependency Check"
+    timeoutInMinutes: 10
+    continueOnError: true
+    pool: sonic-common
+    steps:
+      - script: |
```

Comment:
suggest to use a template yml here to make azure-pipelines.yml clean

Reply:
There is only one step here, so I think there is no necessary to create another template file here?

Reply:
I mean if you need to do some change (like pip install, add library...), you can change in the template file but not azure-pipelines.yml, azure-pipelines.yml file should be more static

Reply:
This makes sence, I will change